README.md

```diff
@@ -1,2 +1,98 @@
 # next-common-files
+
+[travis-url]:https://travis-ci.org/moxystudio/next-common-files
+[travis-image]:http://img.shields.io/travis/moxystudio/next-common-files/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/next-common-files
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-common-files/master.svg
+
 Next.js plugins that insert loaders for common files.
+
+Projects developed with Next.js have to manually insert rules in the configuration file for handling file types that are potentially very common across multiple projects.
+
+These plugins quicken the initial setup of a project by removing that effort from the process, demanding less time from the developer.
+
+## Installation
+``` npm install --save-dev @moxy/next-common-files ```
```

Comment:
Can we use the `sh` notation here? Check https://raw.githubusercontent.com/moxystudio/next-runtime-env/master/README.md

Also add a new line after all headings